vue.config.js

```diff
@@ -0,0 +1,108 @@
+  configureWebpack: () => {
+    const config = {
+      resolve: {
+        alias: {
+          src: resolve(`src`),
+          "@": resolve(`src`),
+          assets: resolve(`src/assets`),
+          scripts: resolve(`src/scripts`),
+          common: resolve(`src/components/common`),
+          transactions: resolve(`src/components/transactions`),
+          govern: resolve(`src/components/govern`),
+          staking: resolve(`src/components/staking`),
+          wallet: resolve(`src/components/wallet`),
+          test: resolve(`test`)
+        },
+        extensions: [`.js`, `.vue`, `.css`]
+      },
+      plugins: [
+        new webpack.DefinePlugin({
+          "process.env": {
+            NODE_ENV: JSON.stringify(process.env.NODE_ENV),
+            RPC: JSON.stringify(process.env.RPC),
+            STARGATE: JSON.stringify(process.env.STARGATE),
+            RELEASE: JSON.stringify(commitHash),
+            GOOGLE_ANALYTICS_UID: JSON.stringify(
+              process.env.GOOGLE_ANALYTICS_UID
+            ),
+            SENTRY_DSN: JSON.stringify(process.env.SENTRY_DSN)
+          }
+        })
+      ]
+    }
+
+    if (!process.env.CI) {
+      config.devServer = {
+        https: {
+          key: fs.readFileSync("./certs/dev.key"),
+          cert: fs.readFileSync("./certs/dev.crt")
+        }
+      }
+    }
+
+    if (process.env.NODE_ENV === `production` && !process.env.E2E_TESTS) {
+      config.plugins.push(
+        // adds the content security policy to the index.html
+        new CSPWebpackPlugin({
+          "object-src": `'none'`,
+          "base-uri": `'self'`,
+          "default-src": `'self'`,
+          "script-src": [
+            `'self'`,
+            `https://app.appzi.io/`,
+            `https://*.lunie.io`
+          ],
+          "worker-src": `'none'`,
+          // 'style-src': production ? `'self'` : `*`, // SECURITY Appzi is applying styles inline, inquired to them already
+          "style-src": [`'self'`, `'unsafe-inline'`],
+          "connect-src": [
+            // third party tools
+            `https://sentry.io`,
+            `https://appzi-collector-b.azurewebsites.net`,
+            `https://keybase.io`,
+            // mainnet
+            `https://stargate.lunie.io`,
+            `wss://rpc.lunie.io:26657`,
+            `https://stargate.cosmos.network`,
+            `wss://rpc.cosmos.network:26657`,
+            // testnet
+            `https://sntajlxzsg.execute-api.eu-central-1.amazonaws.com/`,
+            `wss://test.voyager.ninja:26657`,
+            ...[process.env.STARGATE].filter(x => x !== undefined),
+            ...[process.env.RPC]
+              .filter(x => x !== undefined)
+              .map(x => x.replace("https", "wss"))
+          ],
+          "frame-src": [`'self'`, `https://app.appzi.io/`],
+          "img-src": [
+            `'self'`,
+            `https://www.google-analytics.com/`,
+            `https://s3.amazonaws.com/keybase_processed_uploads/`
```

Comment:
still needed?

Reply:
that's the actual source of the images uploaded by keybase